### ISSUE TITLE
1.0.0

### ISSUE THREAD

**Issue by szepeviktor** [ZeroOneStudio/kenburnsy#2]
Good morning!

Do you plan to implement all those in the readme?

**Comment by szepeviktor** [comment]
@romanenko Maybe you are not watching this repo.

**Comment by romanenko** [comment]
Yes, I would like to implement all plans in the Readme. If you can help on some tasks — it would be great!
